app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,39 @@
+class FriendshipsController < ApplicationController
+  # POST /friendships
+  # POST /friendships.json
+  def create
+    @friendship = current_user.friendships
+      .build(friend_id: params[:friend_id], approved: "false")
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.